apps/sejda-pdf-desktop/sejda-pdf-desktop.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Sejda PDF Desktop'
+description: 'Pleasant and productive PDF software that you''ll love to use'
```

Comment:
Ah I see.

Would you mind using double quotes anyway? "Pleasant and productive PDF software that you'll love to use" just to be sure this doesn't cause any problems down the line.